pages/common/git-rm.md

```diff
@@ -0,0 +1,15 @@
+# git rm
+
+> Remove files from the working tree and from the index
```

Comment:
I don't like using "working tree" like that, since the term isn't self-explanatory. Maybe include "local filesystem" in brackets, or even replace the term with it. And "index" --> "repository index", just to be sure :smile: